.github/workflows/.github/workflows/test-and-compile-dev.yml

```
@@ -0,0 +1,51 @@
+name: CI 
+
+  
+
+on: 
+
+  pull_request: 
+
+    branches: 
+
+      - master 
```

```suggestion
      - main 
```

We use `main` as the default branch.